src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/reference/inputs/EceWorkExperienceReference400HoursEvaluation.vue

```diff
@@ -0,0 +1,225 @@
+<template>
+  <v-row>
+    <v-col cols="12" md="12" lg="12" xl="12">
+      <h2>Work experience hours</h2>
+      <div role="doc-subtitle">Please provide as much detail as possible.</div>
+      <v-row class="mt-5">
+        <v-col cols="12" md="8" lg="6" xl="4">
+          <h2 class="mb-5">Hours</h2>
+          <p class="mb-3">The hours must:</p>
+          <ul class="ml-10">
+            <li>Be related to the field of early childhood education</li>
+            <li>Have been completed between the {{ formatDate(fiveYearsAgo, "LLL d, yyyy") }} and the {{ formatDate(today, "LLL d, yyyy") }}</li>
```

Comment:
I think we have an issue here... 

AC wants us to use 5 years ago IF certificate has expired (I don't see how we determine this yet from the portal invitation) 

if certificate is valid they want it to be within the validity date of the certificate. 

Also should the date that we use be based on today's date or the date the application was submitted. 

https://eccbc.atlassian.net/browse/ECER-1187